examples/samples/build.gradle

```diff
@@ -0,0 +1,15 @@
+plugins {
+    id 'kotlin'
+}
+
+repositories {
+    mavenCentral()
+    mavenLocal()
+}
+
+dependencies {
+    implementation 'dev.mobile:conductor-client:1.0.0-SNAPSHOT'
+    implementation 'dev.mobile:conductor-orchestra:1.0.0-SNAPSHOT'
+    implementation 'dev.mobile:conductor-ios:1.0.0-SNAPSHOT'
```

Comment:
These artifacts are needed to consume conductor in Kotlin/java.

PS: PR is breaking because this 1.0.0-SNAPSHOT was locally published. I will fix it by releasing as soon as PR is approved.